web/themes/joinup/prototype/sass/components/_featured.sass

```diff
@@ -135,11 +135,11 @@
   vertical-align: middle
 
 .featured__form-button
-  +hidden-print
   position: relative
   display: inline-block
   margin: 10px
-  z-index: 99
+  z-index: 999
```

Comment:
I honestly don't see why this would have so high index, i mean it's a button to join/leave collection which is not so important that should be visible even when the modal is open.